### PR TITLE
Llvm submodule

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: scripts/submodules/versions.mk [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+		echo All dependent modules up to date;  \
+	fi
+
+reset:
+	@$(MAKE) validate-versions RESET_VERSIONS=1
+
+__bump-version-%:
+	@if [ "$(REV)" = "" ]; then echo "Usage: make bump-version-$* REV=<ref>"; exit 1; fi
+	python $(SCRIPT) set-rev $* $(REV)
+	@if [ "$(COMMIT)" = "1" ]; then echo "[acceptance-tests] Bump $* to pick up $(REV)." | git commit -F - $(CONFIG); fi
```
Comment (akoeplinger) [inline scripts/submodules/versions.mk]: the `[acceptance-tests]` prefix doesn't make sense in the generalized version. Same on L118 and L123.

File: llvm/SUBMODULES.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+      "name": "llvm", 
+      "url": "git@github.com:mono/llvm.git",
```
Comment (akoeplinger) [inline llvm/SUBMODULES.json]: this should use git:// so it can be cloned anonymously.

**Comment by akoeplinger** [comment]
LGTM apart from the two comments.

**Comment by vargaz** [comment]
@monojenkins build

**Pull request merged**